locksmith/src/controllers/lockController.js

```diff
@@ -36,6 +42,57 @@ const lockGet = async (req, res) => {
   res.json(baseTokenData)
 }
 
+migrateLogEvent.on('migrateLock', async ({ recordId, msg }) => {
+  console.log('migrateLogEvent', recordId, msg)
+  await updateLockMigrationsLog(recordId, msg)
+})
+
+const lockMigrate = async (req, res) => {
+  const { lockAddress } = req.params
+  const unlockVersion = req.query.unlockVersion || 9
+  const chainId = req.chain
+
+  // TODO: make sure lock exists
+  // const databaseLock = await getLockByAddress(lockAddress)
+  // if (!databaseLock) res.send(404)
```

Comment:
I wasnt sure how to handle errors on locksmith API. Should we just return an error code? or is there sth like a formatted error JSON ?

Reply:
We do not have a standard for now. So I think `res.send(404, "Missing lock")` will do it for now